### PR TITLE
NSKOPS-206 create deb package for es 2.2.0

### PR DESCRIPTION
**Pull request by ghostik**
In this PR I added settings for a creation deb package elasticsearch-consul-discovery plugin for es 2.2.0. Note: this plugin work properly only if a jvm has permission = java.lang.reflect.ReflectPermission "suppressAccessChecks", "read".
@sorenmat: cc: @tnn:

### CODE REVIEW

**Comment by mkostrikin** [comment]
How this plugin can be build?
Consider adding README

**Comment by ghostik** [comment]
The plugin can be build using a command:
mvn clean install
dependency: java v. 8

File: src/deb/control/control [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Source: elasticsearch-consul-discovery
+Section: java
+Priority: extra
+Version: [[version]]
+Maintainer: ktu@tradeshift.com
```
Comment (mkostrikin) [inline src/deb/control/control]: Please add your Initials as well,

**Comment by sorenmat** [comment]
why do you need java.security.AllPermission ?

**Comment by ghostik** [comment]
In es 2.2.1 was increased a security policy compared es 2.1.1. (were removed java.lang.reflect.ReflectPermission "suppressAccessChecks"[ #13603] (https://github.com/elastic/elasticsearch/pull/13603). I have changed <java.security.AllPermission> to <java.lang.reflect.ReflectPermission "suppressAccessChecks">. It is more securely. Unfortunately, the plugin don't work with es 2.2.0 without such permissions. I hope, that we can change it in next plugin versions.

**Comment by sorenmat** [comment]
LGTM :+1:

**Comment by mkostrikin** [comment]
:+1:  we tested deb plugin and seems it works.

**Pull request merged**